### PR TITLE
pd-ctl: adjust output format for store-limit

### PR DESCRIPTION
**Pull request by lhy1024**
### What problem does this PR solve?

Issue Number: Ref #4399

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test
- Manual test (add detailed scripts or steps below)

image

### Release note

```release-note
None.
```

### CODE REVIEW

File: tools/pd-ctl/pdctl/command/store_command.go [hunk 1]
```diff
@@ -564,7 +564,7 @@ func storeLimitCommandFunc(cmd *cobra.Command, args []string) {
 		if args[0] == "all" {
 			prefix = storesLimitPrefix
 			if rate > maxStoreLimit {
-				cmd.Printf("rate should less than %f for all\n", maxStoreLimit)
+				cmd.Printf("rate should less than %v for all\n", int(maxStoreLimit))
```
Comment (rleungx) [inline tools/pd-ctl/pdctl/command/store_command.go:567]: We support float for setting the store limit.

Comment (lhy1024) [reply tools/pd-ctl/pdctl/command/store_command.go:567]: How about `rate should less than 200.0 for all`?

Comment (rleungx) [reply tools/pd-ctl/pdctl/command/store_command.go:567]: LGTM

**Review by rleungx — APPROVED** [review]

**Comment by lhy1024** [comment]
PTAL @bufferflies @okJiang

**Review by okJiang — APPROVED** [review]

**Review by niubell — APPROVED** [review]

**Pull request merged**